tests/features/EditorAdminChecks.feature

```diff
@@ -0,0 +1,21 @@
+@api
+Feature: Editor Admin Menu Access Checks
```

Comment:
Nitpick here- this doesnt really have to do with the admin menu, just access to certain pages